apps/console/src/features/users/pages/users.tsx

```diff
@@ -34,11 +34,39 @@ import {
     UIConstants,
     store
 } from "../../core";
+import { GovernanceConnectorInterface, getConnectorCategory } from "../../server-configurations";
 import { deleteUser, getUsersList } from "../api";
 import { AddUserWizard, UsersList, UsersListOptionsComponent } from "../components";
 import { UserManagementConstants } from "../constants";
 import { UserListInterface } from "../models";
 
+/**
+ * Th id of the account manage,ent policy connector category.
+ *
+ * @constant
+ * @type {string}
+ * @default
+ */
+const ACCOUNT_MANAGEMENT_POLICY_CONNECTOR_ID = "QWNjb3VudCBNYW5hZ2VtZW50IFBvbGljaWVz";
```

Comment:
These constants are available in `features/server-configurations/constants/server-configurations-constants.ts`. Shall reuse from there?